test/browser/redux/actions/async.test.js

```diff
@@ -1629,6 +1653,30 @@ describe('Asynchronous Actions', () => {
           }
         ];
         asyncActions.__Rewire__('services', {
+          api: {
+            user: {
+              profile: (cb) => {
+                cb(null, {
+                  fullName: 'Joe',
```

Comment:
Ditto above. A comment of `// not mocking result` or something would be fine if you want.